projects.go

```diff
@@ -63,6 +63,7 @@ type Project struct {
 	ContainerRegistryEnabled                  bool              `json:"container_registry_enabled"`
 	CreatedAt                                 *time.Time        `json:"created_at,omitempty"`
 	LastActivityAt                            *time.Time        `json:"last_activity_at,omitempty"`
+	Links                                     *Links            `json:"_links,omitempty"`
```

Comment:
Please place this below `Statistics` as I try to keep the order of these fields inline with the order used in the docs.

Reply:
sure